tools/shared.py

```diff
@@ -1186,6 +1188,8 @@ def keys(self):
       return self.attrs.keys()
 
     def __getattr__(self, attr):
+      if not self.attrs['STRICT'] and attr in SettingsManager.renamed_settings:
```

Comment:
Why do we need this in getattr? It makes sense in setattr, but we should only be getting properties off a SettingsManager object from Emscripten code, and we shouldn't be using deprecated variables internally, strict mode or no.

Reply:
I removed this.. and added a test.